docker-compose.yml

```diff
@@ -34,3 +44,11 @@ elasticsearch:
 
 redis:
   image: redis
+
+rabbitmq:
+  image: rabbitmq:3-management
+  hostname: olympia
+  name: olympia
+  ports:
+    - "5672:5672"
```

Comment:
I can probably remove it and switch to the regular `rabbitmq` without the management plugin. I had set up the ports while I was debugging this so they might not be needed. I forgot they get exposed on `localhost` on Linux, good call.